handler_test.go

```diff
@@ -271,7 +309,8 @@ func BenchmarkHandlerDispatch(b *testing.B) {
 				Repo: "github.com/yarpc/metrics",
 			},
 		},
-	})
+	}, template.Must(_templates.Clone()))
```

Comment:
ℹ️ Cannot use `getTestTemplates` here because `testing.T != testing.B`.

Reply:
I think `getTestTemplates` can be made to accept a `testing.TB` interface, in which case it'll accept both. 
(Unless there's something testing.T-specific being used in it that I missed.)